pkg/agent/agent.go

```diff
@@ -1132,26 +1135,17 @@ func getRoundInfo(bridgeClient ovsconfig.OVSBridgeClient) types.RoundInfo {
 	return roundInfo
 }
 
-func (i *Initializer) getNodeMTU(transportInterface *net.Interface) (int, error) {
+func (i *Initializer) getInterfaceMTU(transportInterface *net.Interface) (int, error) {
 	if i.mtu != 0 {
 		return i.mtu, nil
 	}
-	mtu := transportInterface.MTU
+	mtu := transportInterface.MTU - i.networkConfig.MTUDeduction
```

Comment:
`i.networkConfig.MTUDeduction` should be deducted after the check in line 1144.

Reply:
Done